docs/api/algorand/transfer-funds.mdx

```diff
@@ -0,0 +1,90 @@
+
+## Responses
+
+### Regular response
+
+|     Name    |         Type        | Description |
+| ----------- | ------------------- | ----------- | 
+| `noSign` 		| `string[]` 					| The array of transactions (in msgpack) that shall not be signed. Together with `toSign` it composes the full array of transactions in the group. |
+| `toSign` 		| `string[]` 					| The array of transactions (in msgpack) that must be signed with the spending key of the `owner` of the DelegatedWallet. Together with the `noSign` transactions it composes the full array of transactions in the group. |
+| `info` 			| `{fee:number}` 			| An object containing `fee`, the amount of DeCash that is being spent on fees in this transaction. | 
+
+### Validation errors
+
+Returned when the parameters are invalid
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400 																															|
+| `errorId` 	| `INVALID_PARAMS` 																									|
+| `message` 	| `Validation errors in CreateDelegatedPayment request.` 						| 
+| `errors` 		| Array of strings containing additional informations 							|
+
+### Sender not opted in 
+
+Returned when the sender has not yet opted into the token that is being sent.
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400 																															|
+| `errorId` 	| `SENDER_NOT_OPTED_IN` 																						|
+| `message` 	| `Sender must opt in to the asset before sending funds.` 					| 
+
+### Sender not enough funds
+
+Returned when the sender does not have enough funds to pay both the transfer of the funds and the fee.
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400 																															|
+| `errorId` 	| `SENDER_NOT_ENOUGH_FUNDS` 																				|
+| `message` 	| `Sender does not have enough funds to send the requested amount.` |
+
+### Receiver not opted in
+
+Returned when a user is sending funds to a non-[DelegatedWallet](#TODO) account that has not yet opted into the token that is being sent.
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400 																															|
+| `errorId`		| `RECEIVER_NOT_OPTED_IN` 																					|
+| `message`		| `Receiver must opt in to the asset before receiving funds.` 			|
+
+## Example Usage
+
+```js
+const response = await fetch('https://api.decash.com/TransferFunds', {
+  method: 'POST',
+  body: JSON.stringify({
+		'assetId': 'usdd', 
```

Comment:
Wondering about the case too

Reply:
Same as previous, fixing it in all pages